contribution.md

```diff
@@ -1,117 +1,135 @@
-# Contribution to Container Images
-
-This document describes rules and best practices for contributing to http://github.com/sclorg/.
+# Contributing to Container Images
```

Comment:
This is good opportunity to explain usage of "ready for review" label in contribution.md. Could you please state it somewhere here and also link a query for searching all open PRs in SCLOrg labeled with this label?

Reply:
https://github.com/search?q=org%3Asclorg+label%3A%22ready+for+review%22+state%3Aopen&type=Issues

This is the query. Could you please double-check if it works as intended?